packages/aws-cdk-lib/aws-scheduler/README.md

```diff
@@ -1,27 +1,64 @@
-# AWS::Scheduler Construct Library
+# Amazon Scheduler Construct Library
 
+Amazon EventBridge Scheduler](https://aws.amazon.com/blogs/compute/introducing-amazon-eventbridge-scheduler/) is a feature from Amazon EventBridge that allows you to create, run, and manage scheduled tasks at scale. With EventBridge Scheduler, you can schedule one-time or recurrently tens of millions of tasks across many AWS services without provisioning or managing underlying infrastructure.
 
-This module is part of the [AWS Cloud Development Kit](https://github.com/aws/aws-cdk) project.
+1. **Schedule**: A schedule is the main resource you create, configure, and manage using Amazon EventBridge Scheduler. Every schedule has a schedule expression that determines when, and with what frequency, the schedule runs. EventBridge Scheduler supports three types of schedules: rate, cron, and one-time schedules. When you create a schedule, you configure a target for the schedule to invoke. 
+2. **Targets**: A target is an API operation that EventBridge Scheduler calls on your behalf every time your schedule runs. EventBridge Scheduler supports two types of targets: templated targets and universal targets. Templated targets invoke common API operations across a core groups of services. For example, EventBridge Scheduler supports templated targets for invoking AWS Lambda Function or starting execution of Step Function state machine. For API operations that are not supported by templated targets you can use customizeable universal targets. Universal targets support calling more than 6,000 API operations across over 270 AWS services.
+3. **Schedule Group**: A schedule group is an Amazon EventBridge Scheduler resource that you use to organize your schedules. Your AWS account comes with a default scheduler group. A new schedule will always be added to a scheduling group. If you do not provide a scheduling group to add to, it will be added to the default scheduling group. You can create up to 500 schedule groups in your AWS account. Groups can be used to organize the schedules logically, access the schedule metrics and manage permissions at group granularity (see details below). Scheduling groups support tagging: with EventBridge Scheduler, you apply tags to schedule groups, not to individual schedules to organize your resources.
 
-```ts nofixture
-import * as scheduler from 'aws-cdk-lib/aws-scheduler';
-```
+This module is part of the [AWS Cloud Development Kit](https://github.com/aws/aws-cdk) project.
 
-<!--BEGIN CFNONLY DISCLAIMER-->
+## Defining a schedule 
 
-There are no official hand-written ([L2](https://docs.aws.amazon.com/cdk/latest/guide/constructs.html#constructs_lib)) constructs for this service yet. Here are some suggestions on how to proceed:
+```ts
```

Comment:
I guess we need a disclaimer here saying that targets doesn't work yet.

Reply:
My initials thought were to add the whole README the information about all upcoming PRs already in this first PR. We could add a notice that this is not all implemented yet but will gradually come in multiple PRs to keep the PR size smaller and reviewable. What do you think @kaizencc? Reworking the README every PR seems a but useless.

Reply:
good with one big disclaimer :) commented that elsewhere but i just saw this